src/utils/frontend-api-client/repairs.js

```diff
@@ -1,7 +1,11 @@
 import axios from 'axios'
 
-export const getRepairs = async () => {
-  const { data } = await axios.get('/api/repairs')
+const PAGE_SIZE = 10
+
+export const getRepairs = async (pageNumber = 1) => {
+  const { data } = await axios.get(
+    `/api/repairs/?PageSize=${PAGE_SIZE}&PageNumber=${pageNumber}`
```

Comment:
We could supply the `PageSize` and `PageNumber` with the `params` key (https://github.com/axios/axios#request-config) which would make this a bit easier to read.

e.g. (untested)

```
await axios.get(
  '/api/repairs/',
  { 
    params:  {
      PageSize: PAGE_SIZE,
      PageNumber: pageNumber
    }
  }
)
```